processor/attributesprocessor/README.md

```diff
@@ -88,6 +88,18 @@ For the `extract` action,
 
  ```
 
+
+For the `convert` action,
```

Comment:
I think we missed the reference to it at the top of the README (in "The supported actions are:" section)